src/pytypes/onefuzztypes/models.py

```diff
@@ -568,6 +568,7 @@ class NodeCommandEnvelope(BaseModel):
 
 class Node(BaseModel):
     timestamp: Optional[datetime] = Field(alias="Timestamp")
+    initialized_at: Optional[datetime]  # Set only once when a node is intialized
```

Comment:
```suggestion

    # Set only once, when a node is initialized.
    initialized_at: Optional[datetime]  
```

Reply:
(fixes typo, formats for breathing room next to the other suggested field coment)